getBoundingClientRect() returns wrong value for tr, td and its descendants for a vertical table

Ahmad-S792 opened the pull request:
#### 1e62e8aa223671686426a66037938de4c25da002
<pre>
getBoundingClientRect() returns wrong value for tr, td and its descendants for a vertical table

getBoundingClientRect() returns wrong value for tr, td and its descendants for a vertical table

<a href="https://bugs.webkit.org/show_bug.cgi?id=170768">https://bugs.webkit.org/show_bug.cgi?id=170768</a>

Reviewed by Ryosuke Niwa.

Patch Authored by Yuki Sekiguchi

RenderTableRow grows toward y axis even if the writing-mode of the table is vertical-rl. The reason is that RenderTableSection::layoutRows() sets the physical x/y instead of the logical x/y. The layout() should set the logical ones.

A RenderTableCell is relative to a RenderTableSection. Since the container of the RenderTableCell is a RenderTableRow, RenderTableCell::offsetFromContainer() should subtract the offset of the RenderTableRow. Since offsetFromContainer() returns the physical offset, it should subtract the flipped x/y of the RenderTableRow. However, it uses the unflipped x/y. Since offsetFromContainer() of the parent returns the flipped offset, RenderTableCell::offsetFromContainer() can use it.

* Source/WebCore/rendering/RenderTableCell.cpp:
(WebCore::RenderTableCell::offsetFromContainer): Used offsetFromContainer() to get the offset of RenderTableRow
* Source/WebCore/rendering/RenderTableSection.cpp:
(WebCore::RenderTableSection::layoutRows): Set the logical x/y instead of the physical x/y
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-child.html: Added Test Case
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-child-expected.txt: Added Test Case Expectations
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-td.html: Added Test Case
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-td-expected.txt: Added Test Case Expectations
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-tr.html: Added Test Case
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-tr-expected.txt: Added Test Case Expectations
* LayoutTests/platform/glib/fast/css/h1-in-section-elements-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/writing-mode/border-styles-vertical-lr-expected.txt: Updated Test Expectations
* LayoutTests/platform/fast/writing-mode/border-styles-vertical-lr-mixed-expected.txt: Updated Test Expectations
* LayoutTests/platform/fast/writing-mode/border-styles-vertical-rl-expected.txt: Updated Test Expectations
* LayoutTests/platform/fast/writing-mode/border-styles-vertical-rl-mixed-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/tables/mozilla_expected_failures/bugs/bug7113-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/text/text-letter-spacing-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/writing-mode/vertical-align-table-baseline-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/tables/mozilla/bugs/bug2962-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/css/h1-in-section-elements-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/overflow/overflow-rtl-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/027-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/028-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/038-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/040-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/rtl-border-collapsing-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/rowspan-paint-order-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/ios/fast/table/table-display-types-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/overflow/overflow-rtl-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/001-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/003-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/rtl-border-collapsing-vertical-expected: Updated Test Expectations
* LayoutTests/fast/table/border-collapsing/collapsed-borders-adjoining-sections-vertical-rl-expected.png: Updated Test Expectations
* LayoutTests/fast/table/border-collapsing/collapsed-borders-adjoining-sections-vertical-rl-expected.txt: Updated Test Expectations
* LayoutTests/fast/table/growCellForImageQuirk-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/auto-with-percent-height-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/001-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/003-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/writing-mode/vertical-align-table-baseline-expected.png: Updated Test Expectations
* LayoutTests/platform/ios/fast/writing-mode/vertical-align-table-baseline-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/css/h1-in-section-elements-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/fast/css/h1-in-section-elements-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-vertical-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/027-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/027-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/028-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/028-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/038-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/038-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/040-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/040-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/auto-with-percent-height-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/auto-with-percent-height-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/fast/repaint/table-writing-modes-h-expected.txt: Updated Test Expectations
* LayoutTests/fast/repaint/table-writing-modes-v-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/tables/027-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/tables/028-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/tables/038-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/tables/040-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/auto-with-percent-height-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/001-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/003-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/rowspan-paint-order-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/table-disply-types-vertical-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/table-disply-types-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/writing-mode/vertical-align-table-baseline-expected: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/table-display-types-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/border-collapsing/rtl-border-collapsing-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/rowspan-paint-order-vertical-expected.txt: Updated Test Expectations

Canonical link: <a href="https://commits.webkit.org/254918@main">https://commits.webkit.org/254918@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/29f5b3de8b69b227f3887e5352965926d348c812

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/90587 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/35171 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/21212 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/99914 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/158262 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/33670 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/28841 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/82947 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/96340 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/96242 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/26796 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/77427 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/26634 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/81574 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/81370 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/69657 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/34768 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/15407 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/32581 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/16385 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/3435 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/36347 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/39308 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/38263 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/35474 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

rniwa commented on Source/WebCore/rendering/RenderTableCell.cpp:
```diff
@@ -353,8 +353,10 @@ LayoutSize RenderTableCell::offsetFromContainer(RenderElement& container, const
-        offset -= parentBox()->locationOffset();
+    if (parent()) {
+        RenderElement* containerOfRow = container.container();
+            offset -= parentBox()->offsetFromContainer(*containerOfRow, point);
```
I don't think it's clear that containerOfRow is always non-null here.

rniwa replied:
Nit: there is an extra indentation for "offset".

Ahmad-S792 replied:
Should I add "comment" or add "releaseNonNull()"? or something else? I can fix "nit". Thanks!

rniwa replied:
No, what I mean is that this seem to be missing a nullptr check.

rniwa reviewed (approved):
r=me

Ahmad-S792 committed:
getBoundingClientRect() returns wrong value for tr, td and its descendants for a vertical table

getBoundingClientRect() returns wrong value for tr, td and its descendants for a vertical table

https://bugs.webkit.org/show_bug.cgi?id=170768

Reviewed by Ryosuke Niwa.

Patch Authored by Yuki Sekiguchi

RenderTableRow grows toward y axis even if the writing-mode of the table is vertical-rl. The reason is that RenderTableSection::layoutRows() sets the physical x/y instead of the logical x/y. The layout() should set the logical ones.

A RenderTableCell is relative to a RenderTableSection. Since the container of the RenderTableCell is a RenderTableRow, RenderTableCell::offsetFromContainer() should subtract the offset of the RenderTableRow. Since offsetFromContainer() returns the physical offset, it should subtract the flipped x/y of the RenderTableRow. However, it uses the unflipped x/y. Since offsetFromContainer() of the parent returns the flipped offset, RenderTableCell::offsetFromContainer() can use it.

* Source/WebCore/rendering/RenderTableCell.cpp:
(WebCore::RenderTableCell::offsetFromContainer): Used offsetFromContainer() to get the offset of RenderTableRow
* Source/WebCore/rendering/RenderTableSection.cpp:
(WebCore::RenderTableSection::layoutRows): Set the logical x/y instead of the physical x/y
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-child.html: Added Test Case
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-child-expected.txt: Added Test Case Expectations
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-td.html: Added Test Case
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-td-expected.txt: Added Test Case Expectations
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-tr.html: Added Test Case
* LayoutTests/fast/dom/Element/getBoundingClientRect-vertical-tr-expected.txt: Added Test Case Expectations
* LayoutTests/platform/glib/fast/css/h1-in-section-elements-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/writing-mode/border-styles-vertical-lr-expected.txt: Updated Test Expectations
* LayoutTests/platform/fast/writing-mode/border-styles-vertical-lr-mixed-expected.txt: Updated Test Expectations
* LayoutTests/platform/fast/writing-mode/border-styles-vertical-rl-expected.txt: Updated Test Expectations
* LayoutTests/platform/fast/writing-mode/border-styles-vertical-rl-mixed-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/tables/mozilla_expected_failures/bugs/bug7113-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/text/text-letter-spacing-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/writing-mode/vertical-align-table-baseline-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/tables/mozilla/bugs/bug2962-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/css/h1-in-section-elements-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/overflow/overflow-rtl-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/027-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/028-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/038-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/040-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/rtl-border-collapsing-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/rowspan-paint-order-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/ios/fast/table/table-display-types-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/overflow/overflow-rtl-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/001-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/003-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/border-collapsing/rtl-border-collapsing-vertical-expected: Updated Test Expectations
* LayoutTests/fast/table/border-collapsing/collapsed-borders-adjoining-sections-vertical-rl-expected.png: Updated Test Expectations
* LayoutTests/fast/table/border-collapsing/collapsed-borders-adjoining-sections-vertical-rl-expected.txt: Updated Test Expectations
* LayoutTests/fast/table/growCellForImageQuirk-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/auto-with-percent-height-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/001-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/003-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/table/border-collapsing/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/ios/fast/writing-mode/vertical-align-table-baseline-expected.png: Updated Test Expectations
* LayoutTests/platform/ios/fast/writing-mode/vertical-align-table-baseline-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/css/h1-in-section-elements-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/fast/css/h1-in-section-elements-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-vertical-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/overflow/overflow-rtl-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/027-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/027-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/028-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/028-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/038-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/038-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/040-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/040-vertical-expected.txt: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/auto-with-percent-height-vertical-expected.png: Updated Test Expectations
* LayoutTest/platform/mac/fast/table/auto-with-percent-height-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/fast/repaint/table-writing-modes-h-expected.txt: Updated Test Expectations
* LayoutTests/fast/repaint/table-writing-modes-v-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/tables/027-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/tables/028-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/tables/038-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/tables/040-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/border-collapsing/002-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/auto-with-percent-height-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/001-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/003-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/004-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/rowspan-paint-order-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/table-disply-types-vertical-expected.png: Updated Test Expectations
* LayoutTests/platform/mac/fast/table/table-disply-types-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/mac/fast/writing-mode/vertical-align-table-baseline-expected: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/border-collapsing/equal-precedence-resolution-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/glib/fast/table/table-display-types-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/border-collapsing/rtl-border-collapsing-vertical-expected.txt: Updated Test Expectations
* LayoutTests/platform/gtk/fast/table/rowspan-paint-order-vertical-expected.txt: Updated Test Expectations

Canonical link: https://commits.webkit.org/254918@main
